pocs/tests/test_pocs.py

```diff
@@ -1,17 +1,37 @@
 import os
 import pytest
 import time
-
-from multiprocessing import Process
+import threading
 
 from astropy import units as u
 
 from pocs import hardware
 from pocs.core import POCS
 from pocs.observatory import Observatory
+from pocs.utils import Timeout
 from pocs.utils.messaging import PanMessaging
 
 
+def wait_for_running(sub, max_duration=90):
+    """Given a message subscriber, wait for a RUNNING message."""
+    timeout = Timeout(max_duration)
+    while not timeout.expired():
+        msg_type, msg_obj = sub.receive_message()
+        if msg_obj and 'RUNNING' == msg_obj.get('message'):
+            return True
+    return False
+
+
+def wait_for_pointing(sub, max_duration=90):
```

Comment:
The MemoryDB PR has a note about perhaps changing which state we check for so we don't run through scheduling.

Reply:
Yup, I've incorporated that in this branch, pushing in a few minutes.